src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,10 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article
+    className={`TodoInfo ${todo.completed ? `TodoInfo--completed` : null}`}
```

Comment:
Instead of rendering 'null' as a class, it would be better to return an empty string ('') when todo is not completed. This will prevent the class 'null' from being applied to the element.